packages/http/src/types.ts

```diff
@@ -292,9 +292,22 @@ export type RouteProducer = (
 export interface HeaderFieldOptions {
   type: "header";
   name: string;
+
+  /**
+   * Equivalent of adding `*` in the path parameter as per [RFC-6570](https://datatracker.ietf.org/doc/html/rfc6570#section-3.2.3)
+   *
+   *  | Style  | Explode | Primitive value = 5 | Array = [3, 4, 5] | Object = {"role": "admin", "firstName": "Alex"} |
+   *  | ------ | ------- | ------------------- | ----------------- | ----------------------------------------------- |
+   *  | simple | false   | `id=5`              | `3,4,5`           | `role,admin,firstName,Alex`                     |
+   *  | simple | true    | `id=5`              | `3,4,5`           | `role=admin,firstName=Alex`                     |
+   *
+   */
+  explode?: boolean;
+
   /**
    * The string format of the array. "csv" and "simple" are used interchangeably, as are
    * "multi" and "form".
+   * @deprecated use explode and `@encode` decorator instead.
```

Comment:
xxx

Reply:
Yes it is being removed. And no what does it mean for header param allow reserve tells uri template you can include / and other uri reserved char this makes not sense for header

Reply:
The format option wasn't used a single time before in the spec repos, so you have some problem with a spec?

Reply:
Got it and thanks for sharing. No problem but we do have some compatible code for `format` interface from compiler and we need to adjust that.